modules/metering-troubleshooting.adoc

```diff
@@ -10,10 +10,173 @@ A common issue with metering is pods failing to start. Pods might fail to start
 [id="metering-not-enough-compute-resources_{context}"]
 == Not enough compute resources
 
-A common issue when installing or running metering is lack of compute resources. Ensure that metering is allocated the minimum resource requirements described in the installation prerequisites.
+A common issue when installing or running metering is a lack of compute resources. As the cluster grows and more reports are created, the `reporting-operator` pod requires more memory. If memory usage reaches the pod limit, the cluster terminates the pod with an `OOMKilled` status. Ensure that metering is allocated the minimum resource requirements described in the installation prerequisites.
+
+NOTE: CPU usage for the `reporting-operator` pod does not increase as the cluster grows.
 
 To determine if the issue is with resources or scheduling, follow the troubleshooting instructions included in the Kubernetes document https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#troubleshooting[Managing Compute Resources for Containers].
 
+To troubleshoot issues due to a lack of compute resources, check the following within the `operator-metering` namespace.
+
+.Procedure
+
+. Check for metering `Report` resources that fail to complete and show the status of `ReportingPeriodUnmetDependencies`:
++
+[source,terminal]
+----
+$ oc get reports
```

Comment:
We may need to call out the need that if you're running commands like `oc get reports` and not supplying the namespace (`-n/--namespace`) parameter, we'll be querying for these resources in whatever the current kube config context is, and not the openshift-metering namespace.

I don't think this is a massive detail, but a quick prerequisite to this procedural section would just be adding the following command:

```oc project openshift-metering```

So you're in the current context when querying for metering-related resources.